homeassistant/components/homekit/accessories.py

```diff
@@ -441,11 +434,8 @@ def async_update_state(self, new_state):
         """
         raise NotImplementedError()
 
-    def call_service(self, domain, service, service_data, value=None):
-        """Fire event and call service for changes from HomeKit."""
-        self.hass.add_job(self.async_call_service, domain, service, service_data, value)
-
-    async def async_call_service(self, domain, service, service_data, value=None):
+    @ha_callback
+    def async_call_service(self, domain, service, service_data, value=None):
         """Fire event and call service for changes from HomeKit.
 
         This method must be run in the event loop.
```

Comment:
These comments are not really necessary anymore if it's `async` or `@callback`

Reply:
Removed the comment